hierarchy/provider.cpp

```diff
@@ -0,0 +1,27 @@
+#include "provider.h"
+#include <iostream>
+
+provider::provider():person()
+{
+}
+
+provider::provider(const provider & toAdd):person(toAdd)
+{
+	consultNum = toAdd.consultNum;
+	weeklyFee = toAdd.weeklyFee;
+}
+
+void provider::display()const
+{
```

Comment:
Unnecessary, we never have to display provider information except in reports and CRUD operations, both of which require specialized displays.